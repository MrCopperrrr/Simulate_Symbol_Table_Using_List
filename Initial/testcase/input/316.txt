INSERT ab string
RPRINT
BEGIN
INSERT ac number
RPRINT
BEGIN
INSERT ad string
RPRINT
END
INSERT ae number
RPRINT
END
INSERT af string
RPRINT