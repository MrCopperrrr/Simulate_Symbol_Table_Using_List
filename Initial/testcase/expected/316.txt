success
ab//0
success
ac//1 ab//0
success
ad//2 ac//1 ab//0
success
ae//1 ac//1 ab//0
success
af//0 ab//0